website/src/models/group.js

```diff
@@ -168,6 +170,81 @@ schema.methods.isMember = function isGroupMember (user) {
   }
 };
 
+schema.methods.startQuest = async function startQuest (user) {
+  if (this.type !== 'party') throw new BadRequest('Must be a party to use this method');
+  if (!this.quest.key) throw new BadRequest('Party does not have a pending quest');
+  if (this.quest.active) throw new BadRequest('Quest is already active');
+
```

Comment:
My intention for these is that they never reach the user. They're sanity checks for programmers to ensure that you don't use `startQuest` if the group is not a party, is not on a quest, or if the quest is already active. These are cases that should never happen in the routes.

In which case, maybe it's better to do InternalServer error so it 500s?

Either that, or just abandon these checks entirely. I think @paglias was in favor. I'm still undecided.